source/connect.txt

```diff
@@ -9,5 +9,147 @@ Connect to a Deployment
+     your connection string from the |service| UI. To learn more, see 
+     :atlas:`Connect to a Cluster <connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`.
+
+Connection Considerations
+-------------------------
+
+Authentication
+~~~~~~~~~~~~~~
+
+To connect to a MongoDB instance requires authentication:
+
+- You can specify the username, password, and authentication database 
+  in the :manual:`connection string </reference/connection-string>`. 
+  For example, to connect to a remote MongoDB instance and
+  authenticate against the ``admin`` database as user ``alice``:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://alice:<password>@mongodb0.examples.com:28015/?authSource=admin"
+
+- You can use the ``--username``, ``--password``, and
+  ``--authenticationDatabase`` command-line options. For example, to
+  connect to a remote MongoDB instance and authenticate against the 
+  ``admin`` database as user ``alice``:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://mongodb0.example.com:28015" --username alice --password <password> --authenticationDatabase admin
+
+  .. note::
+
+     If you don't provide a ``--password`` value, ``mongosh`` prompts 
+     you for one.
+
+Connecting to a Replica Set
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a replica set:
+
+- You can specify the replica set name and members in the
+  :manual:`connection string </reference/connection-string>`:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017/?replicaSet=replA"
+
+- If you are using the :manual:`DNS Seedlist Connection Format 
+  </reference/connection-string/#dns-seedlist-connection-format>`, you 
+  can include the ``+srv`` modifier in your connection string:
+
+  .. code-block:: sh
+
+     mongosh "mongodb+srv://server.example.com/"
+
+  .. note::
+     Using the ``+srv`` connection string modifier automatically sets 
+     the 
+     :manual:`tls option </reference/connection-string/#urioption.tls>` 
+     to ``true`` for the connection. You can 
+     override this behavior by explicitly setting ``tls`` to ``false``.
+
+TLS Connections
+~~~~~~~~~~~~~~~
+
+For ``tls`` connections:
+
+- You can use the 
+  :manual:`tls option </reference/connection-string/#urioption.tls>` to 
+  set ``tls=true`` in the 
+  :manual:`connection string </reference/connection-string>`:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017/?replicaSet=replA&tls=true"
+
+- If you are using the :manual:`DNS Seedlist Connection Format 
+  </reference/connection-string/#dns-seedlist-connection-format>`, the 
+  ``+srv`` connection string modifier automatically sets
+  the ``tls`` option to ``true`` for the connection:
+
+  .. code-block:: sh
+
+     mongosh "mongodb+srv://server.example.com/"
+
+- You can specify the ``--tls`` command-line option.
```

Comment:
Does this work for both the DNS format and non-DNS connection string? If so, we might want to explicitly say so.

Reply:
The DNS format automatically enables tls - I'll add a note.